lib/tasks/jobs.rake

```diff
@@ -75,4 +75,24 @@ namespace :jobs do
                                        thread_grace_period_seconds: args.thread_grace_period_seconds,
                                        publish_metrics: publish_metrics).start_working
   end
+
+  desc 'Generate dummy jobs for tests. Usage: rake jobs:generate_load\'[num,delay]\''
+  task :generate_load, %i[num delay] => :environment do |_t, args|
```

Comment:
I'm not sure if we should have test helpers in the rake tasks. 
There are already scripts like `scripts/seed_service_instances.rb` which are used for similar use cases.
Those can be executed by pasting  the content into a ccng console (`./bin/console`).

Reply:
Converted rake task into generate_jobs script. Works well locally but not yet tested on a bosh VM.